main.go

```diff
@@ -20,10 +20,12 @@ var (
 	port         int
 	servers      string
 	pollInterval time.Duration
+	listenaddr   string
 )
 
 func init() {
 	flag.IntVar(&port, "port", 9120, "The port to serve the endpoint from.")
+        flag.StringVar(&listenaddr, "listenaddr", "", "Address to listen on")
```

Comment:
Sorry for the delay, could you run gofmt on this and I'll merge it in

Reply:
Hey @lucianjon!
Alright, I ran gofmt.